modules/story-budget/story-budget.php

```diff
@@ -328,22 +334,22 @@ function story_budget() {
 			<div class="metabox-holder">
 				<?php
 					echo '<div class="postbox-container columns-number-' . absint( $this->num_columns ) . '">';
-					foreach( (array) $this->terms as $term ) {
-						$this->print_term( $term );
-					}
+				foreach ( (array) $this->terms as $term ) {
+					$this->print_term( $term );
+				}
 
 					echo '</div>';
 				?>
 				<style>
 					<?php
-					  for ( $i = 1; $i <= $this->max_num_columns; ++$i ) {
+					for ( $i = 1; $i <= $this->max_num_columns; ++$i ) {
 						?>
 					.columns-number-<?php echo (int) $i; ?> .postbox {
-						flex-basis: <?php echo  99 / $i ?>%;
+						flex-basis: <?php echo 99 / /* phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped */ $i; ?>%;
```

Comment:
```suggestion
						flex-basis: <?php echo floatval( 99 / $i ); ?>%;
```

should work for sanitization too.

Reply:
Done